src/scripts/main.js

```diff
@@ -1,7 +1,26 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const body = document.querySelector('body');
+  const message = document.createElement('div');
+  const h2 = document.createElement('h2');
+  const p = document.createElement('p');
+
+  message.className = 'notification';
+  message.classList.add(type);
+  message.style.top = posTop + 'px';
+  message.style.right = posRight + 'px';
```

Comment:
Better to use `${…}`:
```suggestion
  message.style.top = `${posTop}px`;
  message.style.right = `${posRight}px`;
```